.github/workflows/linux-workflow.yml

```diff
@@ -108,15 +118,40 @@ jobs:
         env:
           PLATFORM: ${{ matrix.platform }}
           COMPILER: ${{ matrix.compiler }}
+        if: ${{ matrix.detail != '-appimage' }}
         run: ./.github/workflows/scripts/linux/install-packages.sh
 
       - name: Generate CMake
         env:
           PLATFORM: ${{ matrix.platform }}
           COMPILER: ${{ matrix.compiler }}
           ADDITIONAL_CMAKE_ARGS: ${{ matrix.cmakeflags }}
+        if: ${{ matrix.detail != '-appimage' }}
         run: ./.github/workflows/scripts/linux/generate-cmake.sh
 
       - name: Build PCSX2
         working-directory: ./build
+        if: ${{ matrix.detail != '-appimage' }}
         run: ../.github/workflows/scripts/linux/compile.sh
```

Comment:
Being as almost all steps are skipped if we are building the app-image, it would be possible to create a separate workflow that just checkouts the repo / creates the AppImage / uploads it.  Unlike originally, this wouldn't create a bunch of extra builds as that duplicated the entire workflow.

However, this depends on my understanding of how the AppImage is built, which may be wrong!

Reply:
It can be a separate workload. That was my original PR, but it was requested to make it part of the workload to reduce build time, that is also why it is parallel, rather than sequential. (As a matrix element)

Reply:
Well the original implement duplicated the entire workflow file and added AppImage steps.  I'm suggesting a separate workflow file that only has the 2 AppImage steps you added now and remove the steps that you skip if `-appimage` is set.